### PR TITLE
Feat/mempool-pusher

### PR DESCRIPTION
**Pull request by jcnelson**
This PR adds a script in the `tools/` directory that, when run, will periodically scan a local mempool DB and push transactions to a given node's outbound neighbors.

### CODE REVIEW

File: tools/mempool-pusher.sh [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+debug() {
+   if [ $DEBUG -ne 0 ]; then
+      log "$1"
+   fi
+}
+
+# args:
+# $1 mempool path
+# $2 minimum accept time
```
Comment (lgalabru) [inline tools/mempool-pusher.sh:32]: Looks great! Any chance we can add a commented example for each function?

**Review by kantai — APPROVED** [review]

File: tools/mempool-pusher.sh (hunk 1 above)
Comment (jcnelson) [reply tools/mempool-pusher.sh:33]: Added in 23a8adcf6c67cb76d6aaccf00516d7bc272ce11a

Comment (lgalabru) [reply tools/mempool-pusher.sh:33]: Thank you!!

**Review by lgalabru — APPROVED** [review]

**Comment by lgalabru** [comment]
Should we merge to develop instead?

**Comment by jcnelson** [comment]
Sure; done

**Pull request merged**